client/signup/steps/plans/index.jsx

```diff
@@ -282,6 +291,8 @@ export default connect(
 		siteType: getSiteType( state ),
 		hasInitializedSitesBackUrl: hasInitializedSites( state ) ? '/sites/' : false,
 		isMonthlyPricingTest: isTreatmentInMonthlyPricingTest( state ),
+		showTreatmentPlansReorderTest:
+			'treatment' === plans_reorder_abtest_variation || isTreatmentPlansReorderTest( state ),
```

Comment:
Just for curiosity - would the`plans_reorder_abtest_variation` from `signupDependencies` be used during the signup plans step and `isTreatmentPlansReorderTest( state )` be used if the current page is the logged-in plans page?

Reply:
Both are for the signup flow plans step. The former is when a new user signs up and reaches the plans step, the latter is if the user reloads the page on the signup flow.